vue-custom-bootstrap/src/services/api.ts

```diff
@@ -0,0 +1,28 @@
+function fetchApi<T>(url: string, opts = {}): Promise<T> {
+  return fetch(url, opts)
+    .then((response) => {
+      if (!response.ok) {
+        throw new Error(response.statusText);
+      }
+      return response.json() as Promise<T>;
+    })
+    .then((data) => {
+      return data;
+    });
+}
+
+export function getAllForms(): Promise<object> {
+  return fetchApi<{ data: object }>("./forms.json").then(({ data }) => data);
+}
+
+export function saveFormData(
```

Comment:
Would this be for inserts or updates or both? Or just inserts for now

Reply:
Hmm, well it doesn't do anything for now, but I guess it depends how we organize the API endpoints. It's easier if we don't have to check on the front end whether its a new form or an update, so if it works for inserts and updates that would be easier. 

Not sure how much more work that would be though?